index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    
+<header>
+    <!-- <div class="logo">   -->
+      <div class="nav-items logo">
+        <div class="name-text" ><a href="" id="logo">Sunga</a></div>
+    </div>
+    <nav>
+        <div class="nav-items menuicon">
+          <!-- <div class="menuicon"> -->
+        <img class="hamburger" src="./images/Icon - Menu.png" alt="Hamburger menu icon">
+        </div>
+    </nav>
+</header>
+
+<section class="intro">
+    <div class="headline">
+        <h1 class="hello-me">Hey There. I'm Sunga</h1>
+        <h3 class="SD">I'm a Software Developer</h3>
+        <p class="info">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <img class="icons" src="./images/Social Media.png" alt="Social Media icons">
```

Comment:
These social media icons are not exactly for design but they are actually link that will be clickable which will navigate a visitor to your social handles. Hence I recommend downloading this images singly and render them in the page as seen below in order to make it easy for visitors to reach you:

```
<ul>
  <li>
    <a>img here</a>
  </li>
</ul>
```